development/index.md

```diff
@@ -22,7 +22,7 @@ Instructions for creating a release are [here](../development/release.md).
 ### Cadence
 
 FlowForge is released every four weeks, on a Thursday. There are 13 releases each
-year. The initial release was published on 20th of January. This implies releases
+year. The initial release was published on 2022/01/20. This implies releases
```

Comment:
This date format is kind of confusing since it's using `/` which usually notates `DD/MM/YYYY`, could we go with 20th of Janaury, 2022 or if we're sticking with a machine-readable date use the standard `-`'s i.e. `YYYY-MM-DD`